test/e2e/sagemaker/resources/single_variant_endpoint_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: EndpointConfig
+metadata:
+  name: $CONFIG_NAME
+spec:
+  endpointConfigName: $CONFIG_NAME
+  productionVariants:
+    - variantName: variant-1
+      modelName: $CONFIG_NAME-model # test-config-1-model
+      initialInstanceCount: 1
+      instanceType: ml.m5.large
```

Comment:
Lets use the smallest instance type (or cheapest) available

Reply:
my bad, fast launch only works for notebooks